templates/footer-search.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Template for footer search.
+ *
+ * @package WPAMPTheme
+ */
+
+?>
+<!-- Search -->
+<section class='travel-search py4 xs-hide sm-hide relative'>
+	<div class='px1 md-px2 pb1 relative'>
+		<h3 class='travel-search-heading travel-spacing-none h1 bold mb2 center'><?php esc_html_e( 'Have a specific destination in mind?' ); ?></h3>
```

Comment:
Could this include the text domain of `'travel'`, and likewise for the other translation functions in the file?
```php
esc_html_e( 'Have a specific destination in mind?', 'travel' )
```